### PR TITLE
chore(FEC-8975): QoS Enhancements - update docs

### PR DESCRIPTION
**Pull request by RoyBregman**
Added segmentDownloadTime, bandwidth, tabMode, soundMode

### CODE REVIEW

File: docs/kava-parameters.md [hunk 1]
```diff
@@ -188,6 +188,82 @@ Kava parameters are additional data that is sent with Kava event and represent r
...
 ---
 
+<a id="segmentDownloadTime"></a>`segmentDownloadTime` - Longest segment download time in seconds.
+
+- Should be in format of float (second.milliSecond).
+- The value is calculated from the last VIEW event.
+
+---
+
+<a id="bandwidth"></a>`bandwidth` - Average bandwidth in seconds.
```
Comment (OrenMe) [inline docs/kava-parameters.md]:
```suggestion
<a id="bandwidth"></a>`bandwidth` - Average bandwidth in kbps.
```

File: docs/kava-parameters.md [hunk 2]
```diff
@@ -188,6 +188,82 @@ Kava parameters are additional data that is sent with Kava event and represent r
...
+
+- kbps in format of float (second.milliSecond).
```
Comment (OrenMe) [inline docs/kava-parameters.md]: `second.milliSecond` is not relevant here

File: docs/kava-parameters.md [hunk 3]
```diff
@@ -188,6 +188,82 @@ Kava parameters are additional data that is sent with Kava event and represent r
...
+  - 2 - Tab focused
+- The value represents the state when the VIEW event is sent.
+
+---
+
+<a id="soundMode"></a>`soundMode` - Is sound muted or not.
+- Possible values
+  - 1 - Sound off
+  - 2 - Sound on
+- Sound is cosidered off if muted or volume is set to 0
```
Comment (OrenMe) [inline docs/kava-parameters.md]:
```suggestion
- Sound is considered off if muted or volume is set to 0
```

File: docs/kava-parameters.md [hunk 4]
```diff
@@ -188,6 +188,82 @@ Kava parameters are additional data that is sent with Kava event and represent r
...
+---
+
+<a id="manifestDownloadTime"></a>`manifestDownloadTime` - Longest manifest download time in seconds.
+- Should be in format of float (second.milliSecond).
+- The value is calculated from the last VIEW event.
+
+---
+
+<a id="forwardBufferHealth"></a>`forwardBufferHealth` - Ratio between available buffer and target buffer.
+- Should be in format of float (second.milliSecond).
```
Comment (OrenMe) [inline docs/kava-parameters.md]: Doesn't sound like the proper description, it is the percentage

File: docs/kava-parameters.md [hunk 5]
```diff
@@ -188,6 +188,82 @@ Kava parameters are additional data that is sent with Kava event and represent r
...
+---
+
+<a id="targetBuffer"></a>`targetBuffer` - The target buffer of the player.
+- Should be in format of float (second.milliSecond).
+- The value represents the state when the VIEW event is sent.
+
+---
+
+<a id="droppedFramesRatio"></a>`droppedFramesRatio` - Ratio between dropped frames and rendered frames.
+- Should be in format of float (second.milliSecond).
```
Comment (OrenMe) [inline docs/kava-parameters.md]: again, looks like a copy-paste from previous and not relevant for ratio

File: docs/kava-parameters.md [hunk 6]
```diff
@@ -188,6 +188,82 @@ Kava parameters are additional data that is sent with Kava event and represent r
...
+- The value is calculated from the last VIEW event.
+
+---
+
+<a id="playerJSLoadTime"></a>`playerJSLoadTime` - Javascript/player Load time.
+- Should be in format of float (second.milliSecond).
+
+---
+
+<a id="flavorParamsId"></a>`flavorParamsId` - The flavor params Id.
```
Comment (OrenMe) [inline docs/kava-parameters.md]:
```suggestion
<a id="flavorParamsId"></a>`flavorParamsId` - The flavor params Id of the currently playing rendition, only valid for Kaltura live streams.
```

File: docs/kava-parameters.md [hunk 7]
```diff
@@ -188,6 +188,82 @@ Kava parameters are additional data that is sent with Kava event and represent r
...
+
+---
+
+<a id="networkConnectionOverhead"></a>`networkConnectionOverhead` - Max dns+ssl+tcp resolving time of the video segments.
+- Should be in format of float (second.milliSecond).
+- The value is calculated from the last VIEW event.
+
+---
+
+<a id="playerJSLoadTime"></a>`playerJSLoadTime` - Javascript/player Load time.
```
Comment (OrenMe) [inline docs/kava-parameters.md]:
```suggestion
<a id="playerJSLoadTime"></a>`playerJSLoadTime` - the player library load time.
```

File: docs/kava-parameters.md (hunk 4 above)
Comment (RoyBregman) [reply docs/kava-parameters.md]: fixed

**Comment by kaltura-hooks** [comment]
Hi @RoyBregman,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

**Pull request merged**